api_app/connectors_manager/migrations/0031_connector_config_abusesubmitter.py

```diff
@@ -0,0 +1,146 @@
+from django.db import migrations
+from django.db.models.fields.related_descriptors import (
+    ForwardManyToOneDescriptor,
+    ForwardOneToOneDescriptor,
+    ManyToManyDescriptor,
+)
+
+plugin = {
+    "python_module": {
+        "health_check_schedule": None,
+        "update_schedule": None,
+        "module": "abuse_submitter.AbuseSubmitter",
+        "base_path": "api_app.connectors_manager.connectors",
+    },
+    "name": "AbuseSubmitter",
+    "description": "Send email to request the takedown of a malicious domain",
+    "disabled": False,
+    "soft_time_limit": 60,
+    "routing_key": "default",
+    "health_check_status": True,
+    "maximum_tlp": "AMBER",
+    "run_on_failure": True,
+    "model": "connectors_manager.ConnectorConfig",
+}
+
+params = [
+    {
+        "python_module": {
+            "module": "abuse_submitter.AbuseSubmitter",
+            "base_path": "api_app.connectors_manager.connectors",
+        },
+        "name": "sender",
+        "type": "str",
+        "description": "Email sender",
+        "is_secret": False,
+        "required": False,
+    }
+]
+
+values = [
+    {
+        "parameter": {
+            "python_module": {
+                "module": "abuse_submitter.AbuseSubmitter",
+                "base_path": "api_app.connectors_manager.connectors",
+            },
+            "name": "sender",
+            "type": "str",
+            "description": "Email sender",
+            "is_secret": False,
+            "required": False,
+        },
+        "analyzer_config": None,
+        "connector_config": "AbuseSubmitter",
+        "visualizer_config": None,
+        "ingestor_config": None,
+        "pivot_config": None,
+        "for_organization": False,
+        "value": "",
+        "updated_at": "2024-04-22T16:06:41.662930Z",
+        "owner": None,
+    }
+]
```

Comment:
```suggestion
values = [
]
```
same reason as before